migrations/20200323163000-collective-platform-fee.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const OC_FEE_PERCENT = 5;
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Collectives', 'platformFeePercent', {
+      type: Sequelize.INTEGER,
+      defaultValue: OC_FEE_PERCENT,
+    });
```

Comment:
Small detail, is it possible to insert it next to `hostFeePercent`?

Reply:
I checked and it's possible in MySQL but not in Postgres.

Reply:
No, that's not possible in Postgres. Adding column _after_ is a MySQL feature.